README.md

```diff
@@ -179,9 +181,39 @@ schema = {
           }
         }
       };
+
+const values1 = {
+  hospitalProduct: {
+    code: {
+      extrasCode: '12',
+      abc: 'm1'
+    }
+  }
+};
+
+validator.validate(schema, values1).then(result => {
+  console.log(result.valid);  //true
+  console.log(result.values); //{hospitalProduct: {code: {extrasCode: '12', abc: 'm1'}}}
+  console.log(result.errors); //{}
+});
+
+const values2 = {
+  hospitalProduct: {
+    code: {
+      extrasCode: '12',
+      abc: 'toolong'
+    }
+  }
+};
+
+validator.validate(schema, values2).then(result => {
+    console.log(result.valid);  //false
+    console.log(result.values); //{hospitalProduct: {code: {extrasCode: '12' }}}
+    console.log(result.errors); //{hospitalProduct: {code: {abc: 'abc must be less than 2 chars'}}}
+});
 ```
 ##Note about ```filters``` and ```children```
-Filters and children are currently incombatible with each other.
+Filters and children are currently incompatible with each other.
```

Comment:
I originally wrote that part of the read me.
It just means that we currently do not evaluate whole object filters as part of the following execution path:
https://github.com/robertmaldon/schema-validator/blob/76a9e1728130a01bb34c0805bc0fbbdd1ea7ef51/lib/index.js#L81

This probably just needs a re-wording, something identifying the conscious decision to not evaluate filters for child objects and maybe a todo for having children work with filters